demo/App.tsx

```diff
@@ -177,11 +200,14 @@ const App = () => {
                 color="green"
                 testID="addBreadcrumbBtn"
                 accessibilityLabel="addBreadcrumbBtn"
-                onPress={() =>
-                  RaygunClient.recordBreadcrumb('Breadcrumb Message', {
-                    category: 'Simulation',
-                    level: 'debug'
-                  })
+                onPress={() => {
+                  const breadcrumbOption: BreadcrumbOption = {
+                    category: 'Some String you choose',
+                    level: 'debug',
+                    customData: {"Key_6": "My Data is bland"},
+                  }
+                  RaygunClient.recordBreadcrumb('Breadcrumb Message', breadcrumbOption)
```

Comment:
_Future note:_ this API doesn't match the `setUser` API design outlined above. It will be good to consolidate these in a future PR.